### DISCUSSION TITLE
DHT gleich in den Gascounter einbinden.

### DISCUSSION THREAD

**Discussion by Nepomukl** [AK-Homberger/D1Mini-GasCounter#2] [Ideas]
Hallo, es ist doch jetzt für dich ein leichtes einen DHT bzw. BME mit in den Code des Gascounters mit einzubinden.
Das wird doch auch gleich universeller und Leistung ist doch genug vorhanden.
Ich könnte es so gebrauchen ;-)

Besten Dank 

Gruß Andreas

**Comment by AK-Homberger** [comment]
Ich hab das extra standalone gemacht, da man ja üblicherweise in anderen Zimmern die Temperatur und Luftfeuchtigkeit messen möchte und nicht nur im Keller, wo der Gaszähler ist.. 

Aber du kannst es gerne kombinieren. Deshalb ist es ja Open Source.
